Cargo.toml

```diff
@@ -1,12 +1,13 @@
 [package]
 name = "rustposix"
 version = "0.1.0"
-authors = ["Nicholas Smith Renner <nrenner@nyu.edu>"]
+authors = ["Nicholas Smith Renner <nrenner@nyu.edu>", "Jonathan Eli Singer <jes965@nyu.edu>"]
```

Comment:
It'd be nice to have my name in here... no worries if not, though. I understand it's your project :).

Reply:
Yeah of course! Add your name to a commit on this branch. And we'll add it with this merge.